machetli/environments.py

```diff
@@ -63,41 +80,38 @@ def __init__(self, allow_nondeterministic_successor_choice=True,
         self.allow_nondeterministic_successor_choice = \
```

Comment:
I think this should no longer belong to the `Environment`; it is only used by the search.